clients/gtest/matmul_gtest.yaml

```diff
@@ -617,22 +618,40 @@ Tests:
   unit_check: 1
   gpu_arch: '94?'
 
-- name: matmul_f8_dst_f8
+- name: matmul_real_1b_dst_f8
   category: pre_checkin
   function:
     matmul: *real_precisions_1b_dst_f8
   M: [128, 129]
   N: [128, 129]
   K: [128, 129]
-  transA_transB: *deepbench_transA_transB_range
+  transA_transB: *transA_transB_range
+  alpha: 1
+  beta: [ 0.0, 2.0 ]
+  bias_vector: [0, 1]
+  scaleA: [0, 1]
+  scaleB: [0, 1]
+  scaleC: [0, 1]
+  scaleD: [0, 1]
+  bias_type: f32_r
```

Comment:
currently, the default bias type for 8bit out is fp16.

Reply:
Updated as per our discussion in today's meeting. I temporarily disable the bias test for fp8 out.